charts/tidb-cluster/templates/_helpers.tpl

```diff
@@ -73,6 +87,20 @@ config-file: |-
     {{- if .Values.tidb.config }}
 {{ .Values.tidb.config | indent 2 }}
     {{- end -}}
+    {{- if or .Values.enableTLSServer .Values.enableTLSClient }}
+  [security]
+    {{- end -}}
+    {{- if .Values.enableTLSServer }}
+  cluster-ssl-ca = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  cluster-ssl-cert = "/var/lib/tidb-tls/tidb.crt"
+  cluster-ssl-key = "/var/lib/tidb-tls/tidb.key"
```

Comment:
For TiDB, these certs are used to communicate with PD or TiKV or both?

Reply:
ignore me, [this document](https://pingcap.com/docs/v3.0/how-to/secure/enable-tls-between-components/) clarifies it.